modin/pandas/groupby.py

```diff
@@ -711,7 +705,6 @@ def do_relabel(obj_to_relabel):
                     result.columns = new_columns_idx
                     return result
 
-            func_dict = {col: try_get_str_func(fn) for col, fn in func_dict.items()}
```

Comment:
I think we still need this line right - we just have to add the underscore.

Reply:
Nope, it turns out we don't need it